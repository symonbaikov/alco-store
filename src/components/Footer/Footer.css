.footer {
  background-color: #2b2b2b;
  color: #fff;
  padding: 40px 0 20px;
  margin-top: auto;
  width: 100%;
  position: relative;
  bottom: 0;
}

.footer-content {
  max-width: 1500px;
  margin: 0 auto;
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  gap: 40px;
  padding: 0 40px;
}

.footer-section {
  position: relative;
}

.footer-section h3 {
  color: #fff;
  font-family: 'Helvetica Neue', Arial, sans-serif;
  font-size: 13px;
  font-weight: 600;
  text-transform: uppercase;
  margin-bottom: 15px;
  letter-spacing: 0.5px;
}

.footer-section ul {
  list-style: none;
  padding: 0;
  margin: 0;
}

.footer-section ul li {
  margin-bottom: 8px;
}

.footer-section ul li a {
  color: #999;
  text-decoration: none;
  font-size: 13px;
  transition: color 0.3s ease;
}

.footer-section ul li a:hover {
  color: #fff;
}

.contact-section {
  color: #999;
}

.contact-info {
  display: flex;
  flex-direction: column;
  gap: 15px;
}

.contact-item {
  display: grid;
  grid-template-columns: 16px 1fr;
  align-items: center;
  gap: 10px;
  text-decoration: none;
}

.contact-item i {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 16px;
  height: 16px;
  color: #fff;
  font-size: 14px;
}

.contact-item.phone {
  color: #fff;
  font-size: 16px;
  font-weight: 500;
}

.contact-item.email {
  color: #999;
  font-size: 13px;
}

.contact-item.address {
  color: #999;
  font-size: 13px;
}

.working-hours {
  margin-top: 8px;
  padding-left: 26px;
}

.working-hours p {
  margin: 5px 0;
  color: #999;
  font-size: 13px;
}

.working-hours p:first-child {
  margin-bottom: 10px;
}

.footer-bottom {
  margin-top: 30px;
  padding-top: 20px;
  border-top: 1px solid rgba(255, 255, 255, 0.1);
}

.footer-bottom-content {
  max-width: 1500px;
  margin: 0 auto;
  padding: 0 40px;
  display: flex;
  justify-content: space-between;
  align-items: flex-start;
}

.copyright {
  color: #666;
  font-size: 12px;
  max-width: 70%;
}

.copyright p {
  margin: 4px 0;
}

.social-payment {
  display: flex;
  flex-direction: column;
  gap: 15px;
  align-items: flex-end;
}

.social-links {
  display: flex;
  gap: 15px;
}

.social-links a {
  color: #666;
  text-decoration: none;
  font-size: 16px;
  transition: color 0.3s ease;
}

.social-links a:hover {
  color: #fff;
}

.payment-methods {
  display: flex;
  gap: 15px;
  align-items: center;
  margin-top: 20px;
}

.payment-methods span {
  color: #666;
  font-size: 13px;
  transition: color 0.3s ease;
  cursor: default;
}

.payment-methods span:hover {
  color: #fff;
}

@media (max-width: 1024px) {
  .footer-content {
    grid-template-columns: repeat(3, 1fr);
  }

  .contact-section {
    grid-column: span 3;
  }
}

@media (max-width: 768px) {
  .footer-content {
    grid-template-columns: repeat(2, 1fr);
  }

  .contact-section {
    grid-column: span 2;
  }

  .footer-bottom-content {
    flex-direction: column;
    gap: 20px;
    align-items: center;
    text-align: center;
  }

  .social-payment {
    align-items: center;
  }

  .footer-section {
    border-bottom: 1px solid rgba(255, 255, 255, 0.1);
  }

  .footer-section h3 {
    margin: 0;
    padding: 15px 0;
    cursor: pointer;
    display: flex;
    justify-content: space-between;
    align-items: center;
  }

  .footer-section h3::after {
    content: '';
    width: 10px;
    height: 10px;
    border-right: 2px solid #fff;
    border-bottom: 2px solid #fff;
    transform: rotate(45deg);
    transition: transform 0.3s ease;
  }

  .footer-section.active h3::after {
    transform: rotate(-135deg);
  }

  .footer-section ul {
    max-height: 0;
    overflow: hidden;
    transition: max-height 0.3s ease-out;
  }

  .footer-section.active ul {
    max-height: 500px;
    transition: max-height 0.5s ease-in;
  }

  .footer-section ul li:first-child {
    margin-top: 10px;
  }

  .footer-section ul li:last-child {
    margin-bottom: 20px;
  }
}

@media (max-width: 480px) {
  .footer-content {
    grid-template-columns: 1fr;
  }
  
  .contact-section {
    grid-column: span 1;
  }
} 