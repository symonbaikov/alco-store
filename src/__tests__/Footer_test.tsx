import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Footer from '../components/Footer/Footer';

// Мокаем i18next
jest.mock('react-i18next', () => ({
  useTranslation: () => ({
    t: (key: string) => {
      const translations: Record<string, string> = {
        'footer.workingHours.title': 'График за обработка на поръчки:',
        'footer.workingHours.weekdays': 'Пон.-Пет. – от 9:00 до 19:00',
        'footer.workingHours.weekend': 'Съб. – Нед. – от 10:00 до 15:00',
        'footer.catalog': 'КАТАЛОГ',
        'footer.company': 'КОМПАНИЯ',
        'footer.publications': 'ПУБЛИКАЦИИ',
        'footer.help': 'ПОМОЩ',
        'footer.paymentMethods.cash': 'В брой',
        'footer.paymentMethods.mastercard': 'Mastercard',
        'footer.paymentMethods.visa': 'Visa',
        'footer.copyright.rights': '© Онлайн витрина на магазини TRAMONTI.',
        'footer.copyright.ageRestriction': 'Не продаваме алкохол на непълнолетни!',
        'footer.copyright.copyright': 'Разпространението и използването на материали от сайта е забранено!',
        'footer.copyright.resale': 'Магазинът работи само за крайния потребител, препродажбата на алкохол е забранена!',
        'footer.copyright.warning': 'Прекомерната консумация на алкохол вреди на вашето здраве!',
        'footer.links.wine': 'Вино',
        'footer.links.champagne': 'Шампанско',
        'footer.links.cognac': 'Коняк',
        'footer.links.whiskey': 'Уиски',
        'footer.links.vodka': 'Водка',
        'footer.links.liquor': 'Ликьори',
        'footer.links.tequila': 'Текила',
        'footer.links.brandy': 'Бренди',
        'footer.links.beer': 'Бира',
        'footer.links.vermouth': 'Вермути',
        'footer.links.rum': 'Ром',
        'footer.links.about': 'За нас',
        'footer.links.cooperation': 'Сътрудничество',
        'footer.links.rent': 'Наем',
        'footer.links.reviews': 'Отзиви',
        'footer.links.licenses': 'Лицензи',
        'footer.links.documents': 'Документи',
        'footer.links.requisites': 'Реквизити',
        'footer.links.contacts': 'Контакти',
        'footer.links.sommelier': 'Съвети от сомелиер',
        'footer.links.events': 'Събития',
        'footer.links.articles': 'Статии',
        'footer.links.cocktails': 'Коктейли',
        'footer.links.payment-delivery': 'Плащане и доставка',
        'footer.links.return-exchange': 'Връщане и замяна',
        'footer.links.public-offer': 'Публична оферта',
        'footer.links.privacy-policy': 'Политика за поверителност',
        'footer.links.loyalty': 'Програма за лоялност'
      };
      return translations[key] || key;
    },
    i18n: {
      changeLanguage: () => new Promise(() => {}),
    },
  }),
  I18nextProvider: ({ children }: { children: React.ReactNode }) => <>{children}</>,
}));

// Функция для установки размера viewport
function setViewportSize(width: number, height: number) {
  Object.defineProperty(window, 'innerWidth', {
    writable: true,
    configurable: true,
    value: width,
  });
  Object.defineProperty(window, 'innerHeight', {
    writable: true,
    configurable: true,
    value: height,
  });
  window.dispatchEvent(new Event('resize'));
}

describe('Footer Component', () => {
  const renderFooter = () => {
    return render(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>
    );
  };

  beforeEach(() => {
    // Сбрасываем размер viewport перед каждым тестом
    setViewportSize(1920, 1080);
  });

  it('должен корректно отображаться на десктопе', () => {
    const { container } = renderFooter();
    expect(container).toMatchSnapshot();
  });

  it('должен корректно отображаться на планшете', () => {
    setViewportSize(768, 1024);
    const { container } = renderFooter();
    expect(container).toMatchSnapshot();
  });

  it('должен корректно отображаться на мобильном', () => {
    setViewportSize(375, 667);
    const { container } = renderFooter();
    expect(container).toMatchSnapshot();
  });

  it('должен отображать все разделы в футере', () => {
    const { getByText } = renderFooter();
    
    // Проверяем наличие основных разделов
    expect(getByText('КАТАЛОГ')).toBeInTheDocument();
    expect(getByText('КОМПАНИЯ')).toBeInTheDocument();
    expect(getByText('ПУБЛИКАЦИИ')).toBeInTheDocument();
    expect(getByText('ПОМОЩ')).toBeInTheDocument();
  });

  it('должен отображать методы оплаты', () => {
    const { getByText } = renderFooter();
    
    // Проверяем наличие методов оплаты
    expect(getByText('В брой')).toBeInTheDocument();
    expect(getByText('Mastercard')).toBeInTheDocument();
    expect(getByText('Visa')).toBeInTheDocument();
  });

  it('должен отображать рабочие часы', () => {
    const { getByText } = renderFooter();
    
    expect(getByText('График за обработка на поръчки:')).toBeInTheDocument();
    expect(getByText('Пон.-Пет. – от 9:00 до 19:00')).toBeInTheDocument();
    expect(getByText('Съб. – Нед. – от 10:00 до 15:00')).toBeInTheDocument();
  });

  it('должен отображать ссылки в разделе каталога', () => {
    const { getByText } = renderFooter();
    
    expect(getByText('Вино')).toBeInTheDocument();
    expect(getByText('Шампанско')).toBeInTheDocument();
    expect(getByText('Коняк')).toBeInTheDocument();
    expect(getByText('Уиски')).toBeInTheDocument();
  });

  it('должен отображать ссылки в разделе компании', () => {
    const { getByText } = renderFooter();
    
    expect(getByText('За нас')).toBeInTheDocument();
    expect(getByText('Сътрудничество')).toBeInTheDocument();
    expect(getByText('Наем')).toBeInTheDocument();
    expect(getByText('Отзиви')).toBeInTheDocument();
  });

  it('должен отображать ссылки в разделе публикаций', () => {
    const { getByText } = renderFooter();
    
    expect(getByText('Съвети от сомелиер')).toBeInTheDocument();
    expect(getByText('Събития')).toBeInTheDocument();
    expect(getByText('Статии')).toBeInTheDocument();
    expect(getByText('Коктейли')).toBeInTheDocument();
  });

  it('должен отображать ссылки в разделе помощи', () => {
    const { getByText } = renderFooter();
    
    expect(getByText('Плащане и доставка')).toBeInTheDocument();
    expect(getByText('Връщане и замяна')).toBeInTheDocument();
    expect(getByText('Публична оферта')).toBeInTheDocument();
    expect(getByText('Политика за поверителност')).toBeInTheDocument();
  });
}); 