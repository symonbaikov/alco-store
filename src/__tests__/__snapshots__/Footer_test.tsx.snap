// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component должен корректно отображаться на десктопе 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="footer-content"
    >
      <div
        class="footer-section "
      >
        <h3>
          КАТАЛОГ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/wine"
            >
              Вино
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/champagne"
            >
              Шампанско
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cognac"
            >
              Коняк
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/whiskey"
            >
              Уиски
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/vodka"
            >
              Водка
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/liquor"
            >
              Ликьори
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/tequila"
            >
              Текила
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/brandy"
            >
              Бренди
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/beer"
            >
              Бира
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/vermouth"
            >
              Вермути
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/rum"
            >
              Ром
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          КОМПАНИЯ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/about"
            >
              За нас
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cooperation"
            >
              Сътрудничество
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/rent"
            >
              Наем
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/reviews"
            >
              Отзиви
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/licenses"
            >
              Лицензи
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/documents"
            >
              Документи
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/requisites"
            >
              Реквизити
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/contacts"
            >
              Контакти
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          ПУБЛИКАЦИИ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/sommelier"
            >
              Съвети от сомелиер
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/events"
            >
              Събития
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/articles"
            >
              Статии
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cocktails"
            >
              Коктейли
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          ПОМОЩ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/payment-delivery"
            >
              Плащане и доставка
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/return-exchange"
            >
              Връщане и замяна
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/public-offer"
            >
              Публична оферта
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/privacy-policy"
            >
              Политика за поверителност
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/loyalty"
            >
              Програма за лоялност
            </a>
          </li>
        </ul>
      </div>
      <div
        class="contact-section"
      >
        <div
          class="contact-info"
        >
          <a
            class="contact-item phone"
            href="tel:+380979740660"
          >
            <i
              class="fas fa-phone"
            />
            +38 (097) 974-06-60
          </a>
          <a
            class="contact-item email"
            href="mailto:info@tramonti.com.ua"
          >
            <i
              class="fas fa-envelope"
            />
            info@tramonti.com.ua
          </a>
          <div
            class="contact-item address"
          >
            <i
              class="fas fa-map-marker-alt"
            />
            г. Днепр, ул. Нижнеднепровская, 1/Ж
          </div>
          <div
            class="working-hours"
          >
            <p>
              График за обработка на поръчки:
            </p>
            <p>
              Пон.-Пет. – от 9:00 до 19:00
            </p>
            <p>
              Съб. – Нед. – от 10:00 до 15:00
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="footer-bottom"
    >
      <div
        class="footer-bottom-content"
      >
        <div
          class="copyright"
        >
          <p>
            2025
             
            © Онлайн витрина на магазини TRAMONTI.
          </p>
          <p>
            Не продаваме алкохол на непълнолетни!
          </p>
          <p>
            Разпространението и използването на материали от сайта е забранено!
          </p>
          <p>
            Магазинът работи само за крайния потребител, препродажбата на алкохол е забранена!
          </p>
          <p>
            Прекомерната консумация на алкохол вреди на вашето здраве!
          </p>
        </div>
        <div
          class="social-payment"
        >
          <div
            class="social-links"
          >
            <a
              href="https://facebook.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-facebook-f"
              />
            </a>
            <a
              href="https://instagram.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-instagram"
              />
            </a>
            <a
              href="https://youtube.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-youtube"
              />
            </a>
            <a
              href="tel:+380979740660"
            >
              <i
                class="fas fa-phone"
              />
            </a>
          </div>
          <div
            class="payment-methods"
          >
            <span>
              В брой
            </span>
            <span>
              Mastercard
            </span>
            <span>
              Visa
            </span>
          </div>
        </div>
      </div>
    </div>
  </footer>
</div>
`;

exports[`Footer Component должен корректно отображаться на мобильном 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="footer-content"
    >
      <div
        class="footer-section "
      >
        <h3>
          КАТАЛОГ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/wine"
            >
              Вино
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/champagne"
            >
              Шампанско
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cognac"
            >
              Коняк
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/whiskey"
            >
              Уиски
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/vodka"
            >
              Водка
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/liquor"
            >
              Ликьори
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/tequila"
            >
              Текила
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/brandy"
            >
              Бренди
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/beer"
            >
              Бира
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/vermouth"
            >
              Вермути
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/rum"
            >
              Ром
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          КОМПАНИЯ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/about"
            >
              За нас
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cooperation"
            >
              Сътрудничество
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/rent"
            >
              Наем
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/reviews"
            >
              Отзиви
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/licenses"
            >
              Лицензи
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/documents"
            >
              Документи
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/requisites"
            >
              Реквизити
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/contacts"
            >
              Контакти
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          ПУБЛИКАЦИИ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/sommelier"
            >
              Съвети от сомелиер
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/events"
            >
              Събития
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/articles"
            >
              Статии
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cocktails"
            >
              Коктейли
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          ПОМОЩ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/payment-delivery"
            >
              Плащане и доставка
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/return-exchange"
            >
              Връщане и замяна
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/public-offer"
            >
              Публична оферта
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/privacy-policy"
            >
              Политика за поверителност
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/loyalty"
            >
              Програма за лоялност
            </a>
          </li>
        </ul>
      </div>
      <div
        class="contact-section"
      >
        <div
          class="contact-info"
        >
          <a
            class="contact-item phone"
            href="tel:+380979740660"
          >
            <i
              class="fas fa-phone"
            />
            +38 (097) 974-06-60
          </a>
          <a
            class="contact-item email"
            href="mailto:info@tramonti.com.ua"
          >
            <i
              class="fas fa-envelope"
            />
            info@tramonti.com.ua
          </a>
          <div
            class="contact-item address"
          >
            <i
              class="fas fa-map-marker-alt"
            />
            г. Днепр, ул. Нижнеднепровская, 1/Ж
          </div>
          <div
            class="working-hours"
          >
            <p>
              График за обработка на поръчки:
            </p>
            <p>
              Пон.-Пет. – от 9:00 до 19:00
            </p>
            <p>
              Съб. – Нед. – от 10:00 до 15:00
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="footer-bottom"
    >
      <div
        class="footer-bottom-content"
      >
        <div
          class="copyright"
        >
          <p>
            2025
             
            © Онлайн витрина на магазини TRAMONTI.
          </p>
          <p>
            Не продаваме алкохол на непълнолетни!
          </p>
          <p>
            Разпространението и използването на материали от сайта е забранено!
          </p>
          <p>
            Магазинът работи само за крайния потребител, препродажбата на алкохол е забранена!
          </p>
          <p>
            Прекомерната консумация на алкохол вреди на вашето здраве!
          </p>
        </div>
        <div
          class="social-payment"
        >
          <div
            class="social-links"
          >
            <a
              href="https://facebook.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-facebook-f"
              />
            </a>
            <a
              href="https://instagram.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-instagram"
              />
            </a>
            <a
              href="https://youtube.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-youtube"
              />
            </a>
            <a
              href="tel:+380979740660"
            >
              <i
                class="fas fa-phone"
              />
            </a>
          </div>
          <div
            class="payment-methods"
          >
            <span>
              В брой
            </span>
            <span>
              Mastercard
            </span>
            <span>
              Visa
            </span>
          </div>
        </div>
      </div>
    </div>
  </footer>
</div>
`;

exports[`Footer Component должен корректно отображаться на планшете 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="footer-content"
    >
      <div
        class="footer-section "
      >
        <h3>
          КАТАЛОГ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/wine"
            >
              Вино
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/champagne"
            >
              Шампанско
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cognac"
            >
              Коняк
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/whiskey"
            >
              Уиски
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/vodka"
            >
              Водка
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/liquor"
            >
              Ликьори
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/tequila"
            >
              Текила
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/brandy"
            >
              Бренди
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/beer"
            >
              Бира
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/vermouth"
            >
              Вермути
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/rum"
            >
              Ром
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          КОМПАНИЯ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/about"
            >
              За нас
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cooperation"
            >
              Сътрудничество
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/rent"
            >
              Наем
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/reviews"
            >
              Отзиви
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/licenses"
            >
              Лицензи
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/documents"
            >
              Документи
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/requisites"
            >
              Реквизити
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/contacts"
            >
              Контакти
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          ПУБЛИКАЦИИ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/sommelier"
            >
              Съвети от сомелиер
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/events"
            >
              Събития
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/articles"
            >
              Статии
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/cocktails"
            >
              Коктейли
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer-section "
      >
        <h3>
          ПОМОЩ
        </h3>
        <ul>
          <li>
            <a
              data-discover="true"
              href="/payment-delivery"
            >
              Плащане и доставка
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/return-exchange"
            >
              Връщане и замяна
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/public-offer"
            >
              Публична оферта
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/privacy-policy"
            >
              Политика за поверителност
            </a>
          </li>
          <li>
            <a
              data-discover="true"
              href="/loyalty"
            >
              Програма за лоялност
            </a>
          </li>
        </ul>
      </div>
      <div
        class="contact-section"
      >
        <div
          class="contact-info"
        >
          <a
            class="contact-item phone"
            href="tel:+380979740660"
          >
            <i
              class="fas fa-phone"
            />
            +38 (097) 974-06-60
          </a>
          <a
            class="contact-item email"
            href="mailto:info@tramonti.com.ua"
          >
            <i
              class="fas fa-envelope"
            />
            info@tramonti.com.ua
          </a>
          <div
            class="contact-item address"
          >
            <i
              class="fas fa-map-marker-alt"
            />
            г. Днепр, ул. Нижнеднепровская, 1/Ж
          </div>
          <div
            class="working-hours"
          >
            <p>
              График за обработка на поръчки:
            </p>
            <p>
              Пон.-Пет. – от 9:00 до 19:00
            </p>
            <p>
              Съб. – Нед. – от 10:00 до 15:00
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="footer-bottom"
    >
      <div
        class="footer-bottom-content"
      >
        <div
          class="copyright"
        >
          <p>
            2025
             
            © Онлайн витрина на магазини TRAMONTI.
          </p>
          <p>
            Не продаваме алкохол на непълнолетни!
          </p>
          <p>
            Разпространението и използването на материали от сайта е забранено!
          </p>
          <p>
            Магазинът работи само за крайния потребител, препродажбата на алкохол е забранена!
          </p>
          <p>
            Прекомерната консумация на алкохол вреди на вашето здраве!
          </p>
        </div>
        <div
          class="social-payment"
        >
          <div
            class="social-links"
          >
            <a
              href="https://facebook.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-facebook-f"
              />
            </a>
            <a
              href="https://instagram.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-instagram"
              />
            </a>
            <a
              href="https://youtube.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                class="fab fa-youtube"
              />
            </a>
            <a
              href="tel:+380979740660"
            >
              <i
                class="fas fa-phone"
              />
            </a>
          </div>
          <div
            class="payment-methods"
          >
            <span>
              В брой
            </span>
            <span>
              Mastercard
            </span>
            <span>
              Visa
            </span>
          </div>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
